lib/main.coffee

```diff
@@ -1,11 +1,30 @@
 {CompositeDisposable} = require('atom')
 
+# The default grammar scopes checked by this linter.
+defaultGrammarScopes = [
+  'text.html.basic'
+]
+
 module.exports =
   config:
     executablePath:
       default: 'tidy'
       title: 'Full path to the `tidy` executable'
       type: 'string'
+    customGrammarScopes:
+      default: []
+      title: 'Custom Grammar Scopes'
+      description: 'A list of extra grammar scopes to lint with Tidy.<br/><br/>
+        By default, this package only lints HTML scopes known to work cleanly
+        with Tidy. If you know of any HTML variants that Tidy works with without
+        producing spurious errors, please
+        [let us know](https://github.com/AtomLinter/linter-tidy/issues)
+        so that we may improve the default list.<br/><br/>
+        The following grammar scopes are linted by default and do not need to
```

Comment:
I would cut out this line, it can only be changed on reloading the package, and as the configuration should really be moved into the `package.json` where it would be static anyway, this isn't the best thing to stick in here.